diff --git a/Test.java b/Test.java
index 5634b6b..3a74722 100644
--- a/Test.java
+++ b/Test.java
@@ -5,12 +5,11 @@ public class Test {
         int[] a2 = new int[]{
                 1,
                 2  // violation without filter
-                ,};
+                };
 
         int[] a3 = new int[]{
                 1,
                 2  // violation without filter
-                ,
         };
     }
 }
