diff --git a/Test.java b/Test.java
index 35c9af9..369bbd3 100644
--- a/Test.java
+++ b/Test.java
@@ -4,19 +4,20 @@ public class Test {
     public void test() {
         {
             try {  // violation
+
             }
             finally {  // violation without filter
             }
             try {  // violation
-                // something
+
             }
             finally {  // violation
-                // something
             }
             try {
                 ; // something
             }
             finally {
+                System.out.println();
                 ; // statement
             }
         }
