diff --git a/Test.java b/Test.java
index 729a114..877bf70 100644
--- a/Test.java
+++ b/Test.java
@@ -18,6 +18,9 @@ public class Test {  // violation without filter
             System.identityHashCode("running");
             System.identityHashCode("running");
             System.identityHashCode("running");
+            System.identityHashCode("running");
+            System.identityHashCode("running");
+            System.identityHashCode("running");
         }
     };
 }
