diff --git a/Test.java b/Test.java
index 5189eae..36d9cb3 100644
--- a/Test.java
+++ b/Test.java
@@ -2,7 +2,7 @@ package TreeWalker.coding.SimplifyBooleanReturn;
 
 public class Test {
     public boolean test(boolean cond) {
-        if (cond) {
+        if (cond)  {  // violation without filter
             return true;
         }
         else {
@@ -10,3 +10,14 @@ public class Test {
         }
     }
 }
+
+class Test1 {
+    public boolean test1(boolean cond1) {
+        if (cond1) {  // violation
+            return true;
+        }
+        else {
+            return false;
+        }
+    }
+}
