diff --git a/Test.java b/Test.java
index aea01e2..5982414 100644
--- a/Test.java
+++ b/Test.java
@@ -2,7 +2,6 @@ package Checker.FileTabCharacter;
 
 public class Test {
     // Long line ----------------------------------------------------------------
-    // Contains a tab ->	<- //warn
 
 
 
@@ -17,6 +16,7 @@ public class Test {
 
 
     // Long line ----------------------------------------------------------------
+    // Contains a tab ->	<- //warns
     // Long line ----------------------------------------------------------------
     // Contains a tab ->	<- //warns
 
