--- Input_original.java	2020-07-12 17:01:49.228134700 +0100
+++ Input.java	2020-07-12 16:48:53.828844600 +0100
@@ -1,9 +1,14 @@
 class Input {
     public void a() {
-        System.out.print("a");
+        System.out.print("changed"); // no violation, violation as it changed in patch
     }
 
     public boolean b() {
+        System.out.print("b"); // violation as it added in patch
         return false;
     }
+
+    public void c() {
+        System.out.print("c"); // violation as it added in patch
+    }
 }
