diff --git a/src/main/java/Checker/LineLength.java b/src/main/java/Checker/LineLength.java
index 5108b81..f6e3b37 100644
--- a/LineLength.java
+++ b/LineLength.java
@@ -5 +5 @@ public class LineLength {
-        System.out.println("hhhhh");
+        System.out.println("hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh");
@@ -9,0 +10 @@ public class LineLength {
+        System.out.println("hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh");
