diff --git a/Test.java b/Test.java
index 3ed22b2..b386e89 100644
--- a/Test.java
+++ b/Test.java
@@ -6,11 +6,12 @@ public class Test {
 class Test1 {
     private String s; // OK
     class InnerTest1 {}
-    public void test() {} // violation without filter
+    public void test1() {} // violation without filter
 }
 
 class Test2 {
     private String s; // OK
     class InnerTest1 {}
     public void test() {} // violation without filter
+    public void test1() {} // violation without filter
 }
