diff --git a/Test.java b/Test.java
index bc3ea95..fe30367 100644
--- a/Test.java
+++ b/Test.java
@@ -3,7 +3,7 @@ package TreeWalker.InterfaceIsType;
 public class Test {
 }
 
-interface Test1 {
+interface Test0 {  // violation without filter
     int a = 32;
 }
 
@@ -15,3 +15,7 @@ interface Test2 { // violation without filter
 interface Test3 { // violation without filter
     int a = 3;
 }
+
+interface Test4 { // violation
+    int a = 3;
+}
