diff --git a/Test.java b/Test.java
index 630cdd9..b0ed5da 100644
--- a/Test.java
+++ b/Test.java
@@ -5,9 +5,6 @@ public class Test {
         return false;
     }
 
-    public boolean equals(Object i) {
-        return false;
-    }
 }
 
 class Test2 {
@@ -15,12 +12,18 @@ class Test2 {
         return false;
     }
 
-    public boolean equals(Object i) {
-        return false;
-    }
+//    public boolean equals(Object i) {
+//        return false;
+//    }
 }
 
 
 
 
 
+
+class Test3 {
+    public boolean equals(Test i) {  // violation without filter
+        return false;
+    }
+}
