diff --git a/src/main/java/Checker/FileTabCharacter/Test.java b/src/main/java/Checker/FileTabCharacter/Test.java
index 70fc74a..dc960b0 100644
--- a/src/main/java/Checker/FileTabCharacter/Test.java
+++ b/src/main/java/Checker/FileTabCharacter/Test.java
@@ -5 +5 @@ public class Test {
-    // Contains a tab ->	<- //warns
+    // Contains a tab ->	<- //warn
@@ -21,0 +22,2 @@ public class Test {
+    // Contains a tab ->	<- //warns
+    // Contains trailing whitespace ->
