diff --git a/Test.java b/Test.java
index aa44011..63f6e7c 100644
--- a/Test.java
+++ b/Test.java
@@ -2,7 +2,6 @@ package Checker.FileTabCharacter;
 
 public class Test {
     // Long line ----------------------------------------------------------------
-    // Contains a tab ->	<- //warn
 
 
 
@@ -11,6 +10,7 @@ public class Test {
     // Long line ----------------------------------------------------------------
     // Long line ----------------------------------------------------------------
     // Contains a tab ->	<- //warns
+    // Contains a tab ->	<- //warns  // violation without filter
 
 
 }
