diff --git a/Test.java b/Test.java
index 63f6e7c..d57b91c 100644
--- a/Test.java
+++ b/Test.java
@@ -10,7 +10,7 @@ public class Test {
     // Long line ----------------------------------------------------------------
     // Long line ----------------------------------------------------------------
     // Contains a tab ->	<- //warns
-    // Contains a tab ->	<- //warns
+    // Contains a tab ->	<- //warn
 
 
 }
