diff --git a/Test.java b/Test.java
index 7dd3e99..5982414 100644
--- a/Test.java
+++ b/Test.java
@@ -17,7 +17,6 @@ public class Test {
 
     // Long line ----------------------------------------------------------------
     // Contains a tab ->	<- //warn
-    // Contains a tab ->	<- //warn
     // Long line ----------------------------------------------------------------
     // Contains a tab ->	<- //warns
 
