diff --git a/Test.java b/Test.java
index d57b91c..41cc056 100644
--- a/Test.java
+++ b/Test.java
@@ -9,7 +9,6 @@ public class Test {

     // Long line ----------------------------------------------------------------
     // Long line ----------------------------------------------------------------
-    // Contains a tab ->	<- //warns
     // Contains a tab ->	<- //warn


diff --git a/Test2.java b/Test2.java
index 6055de3..b4ea012 100644
--- a/Test2.java
+++ b/Test2.java
@@ -1,4 +1,5 @@
 package Checker.FileTabCharacter;

 public class Test2 {
+    // Contains a tab ->	<- //warn  // violation without filter
 }
