diff --git a/Test.java b/Test.java
index 5982414..e7e751e 100644
--- a/Test.java
+++ b/Test.java
@@ -16,7 +16,6 @@ public class Test {
 
 
     // Long line ----------------------------------------------------------------
-    // Contains a tab ->	<- //warn
     // Long line ----------------------------------------------------------------
     // Contains a tab ->	<- //warns
 
diff --git a/Test2.java b/Test2.java
index 6055de3..b4ea012 100644
--- a/Test2.java
+++ b/Test2.java
@@ -1,4 +1,5 @@
 package Checker.FileTabCharacter;
 
 public class Test2 {
+    // Contains a tab ->	<- //warns
 }
