diff --git a/Test.java b/Test.java
index 7b3fdb2..1b589c2 100644
--- a/Test.java
+++ b/Test.java
@@ -1,6 +1,7 @@
 package TreeWalker.coding.HiddenField;
 
 public class Test {
+    private int test;
 
     private void addTest(int test)  // violation without filter
     {
@@ -9,6 +10,6 @@ public class Test {
 
     private void foo()
     {
-        final int test1 = 1;
+        final int test = 1;  // violation without filter
     }
 }
