diff --git a/src/main/java/Test2.java b/src/main/java/Test2.java
index 91aef3b..aba6949 100644
--- a/src/main/java/Test2.java
+++ b/src/main/java/Test2.java
@@ -2,6 +2,14 @@ public class Test2 {
     public void test1() {

     }
+
+    public void test2() {
+
+    }
+
+    public void test3() {
+
+    }
 }


@@ -9,5 +17,7 @@ public class Test2 {
 class Test22 {
     public void test1() {
         System.out.println();
+        System.out.println();
+        System.out.println();
     }
 }
diff --git a/src/main/java/Test3.java b/src/main/java/Test3.java
index 65cab73..4df8f12 100644
--- a/src/main/java/Test3.java
+++ b/src/main/java/Test3.java
@@ -2,6 +2,14 @@ public class Test3 {
     public void test1() {

     }
+
+    public void test2() {
+
+    }
+
+    public void test3() {
+
+    }
 }


@@ -15,5 +23,7 @@ public class Test3 {
 class Test33 {
     public void test1() {
         System.out.println();
+        System.out.println();
+        System.out.println();
     }
 }