diff --git a/Test.java b/Test.java
index 9259efe..48c43f2 100644
--- a/Test.java
+++ b/Test.java
@@ -2,7 +2,7 @@ package TreeWalker.ParameterNumber;
 
 public class Test {
     public void test(int aArg1, int aArg2, int aArg3, int aArg4, int aArg5,  // violation context
-                     int aArg6, int aArg7, int aArg8, int aArg9) {
+                     int aArg6, int aArg7, int aArg8, int aArg9, int aArg10) {
 
     }
 }
