diff --git a/Test.java b/Test.java
index 35c9af9..d1e35a0 100644
--- a/Test.java
+++ b/Test.java
@@ -4,13 +4,14 @@ public class Test {
     public void test() {
         {
             try {  // violation without filter
+
             }
             finally {  // violation without filter
             }
             try {  // violation without filter
                 // something
             }
-            finally {
+            finally  {  // violation without filter
                 // something
             }
             try {
@@ -19,6 +20,11 @@ public class Test {
             finally {
                 ; // statement
             }
+            try {  // violation
+
+            }
+            finally {  // violation
+            }
         }
     }
 }
