diff --git a/Test.java b/Test.java
index 7e1625a..6900690 100644
--- a/Test.java
+++ b/Test.java
@@ -1 +1 @@
-package Checker.RegexpOnFilename;
+package Checker.RegexpOnFilenames;  // violation without filter
diff --git a/Test2.java b/Test2.java
index 1003830..82306c9 100644
--- a/Test2.java
+++ b/Test2.java
@@ -3,0 +4,3 @@ public class Test2 {
+    public static void main(String[] args) {
+
+    }
diff --git a/Test3.java b/Test3.java
new file mode 100644
index 0000000..86748d4
--- /dev/null
+++ b/Test3.java
@@ -0,0 +1,4 @@
+package Checker.RegexpOnFilename;  // violation without filter
+
+public class Test3 {
+}
