diff --git a/Test.java b/Test.java
index 1e63b38..aebed33 100644
--- a/Test.java
+++ b/Test.java
@@ -10,7 +10,7 @@ public class Test {
         // body
     }
 
-    public void myFunc() throws CloneNotSupportedException,
+    public void myFunc() throws  CloneNotSupportedException,  // violation without filter
             StringIndexOutOfBoundsException,
             StringIndexOutOfBoundsException,
             IllegalStateException,
@@ -26,4 +26,12 @@ public class Test {
         // body
         System.out.println();
     }
+
+    public void myFunc2() throws CloneNotSupportedException,  // violation without filter
+            StringIndexOutOfBoundsException,
+            StringIndexOutOfBoundsException,
+            IllegalStateException,
+            NullPointerException {
+        // body
+    }
 }
